tests/integration/components/frost-bunsen-form/renderers/when-test.js

```diff
@@ -83,7 +83,7 @@ describe('Integration: Component / frost-bunsen-form / renderer / when', functio
     it('renders as expected', function () {
       expectCollapsibleHandles(0)
       expectBunsenWhenRendererWithState('foo', {label: 'FooBar Baz'})
-      expectOnValidationState(ctx, {count: 2})
+      expectOnValidationState(ctx, {count: 4})
```

Comment:
I think this means that 2 more validations resulted from this change? How? I didn't see any change that would have contributed to this.